libs/ui-lib/src/index.ts

```diff
@@ -1,3 +1,4 @@
+export * from './lib/custom-file-upload/custom-file-upload';
```

Comment:
Please move this to mull-ui's component folder. As we discussed, ui-lib will be removed around the end of the sprint